internal/testsetups/simple/api/simple_issuer_types.go

```diff
@@ -46,7 +46,7 @@ func (vi *SimpleIssuer) GetStatus() *v1alpha1.IssuerStatus {
 }
 
 func (vi *SimpleIssuer) GetIssuerTypeIdentifier() string {
-	return "simpleissuers.issuer.cert-manager.io"
+	return "simpleissuers.testing.cert-manager.io"
```

Comment:
We were using "simpleissuers.issuer.cert-manager.io", which does not match our group name.